src/appshell/internal/applicationactioncontroller.cpp

```diff
@@ -21,14 +21,72 @@
 #include <QCoreApplication>
 
 using namespace mu::appshell;
-using namespace mu::actions;
+using namespace mu::framework;
 
 void ApplicationActionController::init()
 {
     dispatcher()->reg(this, "quit", this, &ApplicationActionController::quit);
+
+    dispatcher()->reg(this, "about-qt", this, &ApplicationActionController::openAboutQtDialog);
+
+    dispatcher()->reg(this, "online-handbook", this, &ApplicationActionController::openOnlineHandbookPage);
+    dispatcher()->reg(this, "ask-help", this, &ApplicationActionController::openAskForHelpPage);
+    dispatcher()->reg(this, "report-bug", this, &ApplicationActionController::openBugReportPage);
+    dispatcher()->reg(this, "leave-feedback", this, &ApplicationActionController::openLeaveFeedbackPage);
+
+    dispatcher()->reg(this, "revert-factory", this, &ApplicationActionController::revertToFactorySettings);
 }
 
 void ApplicationActionController::quit()
 {
     QCoreApplication::quit();
 }
+
+void ApplicationActionController::openAboutQtDialog()
+{
+    QApplication::aboutQt();
+}
+
+void ApplicationActionController::openOnlineHandbookPage()
+{
+    QString currentLanguageCode = languagesService()->currentLanguage().val.code;
```

Comment:
1. better to implement a separate method for getting current language code:

currentLanguageCode() {
return languagesService()->currentLanguage().val.code
}

2. I think it is better to remove languageCode from configuration methods and get it dirrectly in their implementation. I doubt that somewhere we will use any of these methods with non-current language code

Reply:
moved getting language code to configuration